src/common/file_entry.rs

```diff
@@ -1,6 +1,6 @@
 use std::fmt;
 use std::fs;
-use std::path::PathBuf;
+use std::path::{Path, PathBuf};
```

Comment:
It looks like it's the convention to put each import on its own line, should I split this up?

Reply:
No, everything is fine 👌